docs/redirects.yaml

```
@@ -82,7 +82,11 @@
 
 - type: page
   from_url: /api/augur.export_v1.html
-  to_url: /api/developer/augur.export_v1.html
+  to_url: /api/developer/augur.export_v2.html
+
+- type: page
+  from_url: /api/augur.export_v1.html
+  to_url: /api/developer/augur.export_v2.html
```

Looks like a duplicate entry as above, did you mean 

```suggestion
- type: page
  from_url: /api/developer/augur.export_v1.html
  to_url: /api/developer/augur.export_v2.html
```

---

Totally! I've also updated the two redirects to point to the migration guide (thanks @victorlin!)